01-filedir.md

```diff
@@ -336,7 +336,10 @@ As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
-but we'll see some uses for it soon.
+but we'll see some uses for it soon.  
+Finally, we also see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). There may also be similar files called `.bashrc` or `.bash_login`. For this lesson material it does not contain any settings.
```

Comment:
I agree with @lexnederbragt here, the earlier paragraph explicitly states that `-a` reveals files with a leading dot, so I'm fine with what you had.